### PR TITLE
buffs vault door and disables weather

### PR DESCRIPTION
**Pull request by LetterN**

### CODE REVIEW

File: code/game/vault_door.dm [hunk 1]
```diff
@@ -27,48 +30,70 @@
...
-//� �� ���� ������������ ��� ������  - Google translate tells me that from Russian to english this is "Do not move your arms around this way." so dont.
+/obj/structure/vaultdoor/attackby(obj/item/I, mob/living/user, params)
+	add_fingerprint(user)
+	if(istype(I, /obj/item/weldingtool) && user.a_intent == INTENT_HELP)
+		if(obj_integrity < max_integrity)
+			if(!I.tool_start_check(user, amount=0))
+				return
+
+			to_chat(user, "<span class='notice'>You begin repairing [src]...</span>")
+			if(I.use_tool(src, user, 40, volume=50))
```
Comment (ma44) [inline code/game/vault_door.dm]: I don't think there's any welder you can get normally that has a volume of 50 unless you somehow get a 80 volume industrial welding tool

File: code/game/vault_door.dm [hunk 2]
```diff
@@ -27,48 +30,70 @@
...
 	set_opacity(0)
-	src.density = 0
-	is_busy = 0
+	src.density = FALSE
+	is_busy = FALSE
+	is_open = TRUE
 
 /obj/structure/vaultdoor/proc/close()
-	is_busy = 1
+	is_busy = TRUE
```
Comment (AndrewL97) [inline code/game/vault_door.dm]: Not for this pull (unless you're keen)
but we should also do a loop on everything on the door turf and gib/damage if it shuts on them, to make it a bit more risky trying to slip through as it shuts

File: code/game/vault_door.dm (hunk 1 above)
Comment (AndrewL97) [reply code/game/vault_door.dm]: I think hacked lathes spit these out?

If this is an issue we can probably just make this a gradual thing. Like each weld uses 10 fuel and increases integrity (capped) by max_integrity/5 or something

File: code/modules/clothing/under/f13.dm [hunk 3]
```diff
@@ -188,7 +195,7 @@
 
 /obj/item/clothing/under/f13/cowboyg
 	name = "Cowboy outfit"
-	desc = "A dirt Grey cowboy outfit. Specially usefull for herding brahmins."
+	desc = "A dirt grey cowboy outfit. Specially usefull for herding brahmins."
```
Comment (AndrewL97) [inline code/modules/clothing/under/f13.dm]: gey*

File: code/game/vault_door.dm (hunk 1 above)
Comment (AndrewL97) [reply code/game/vault_door.dm]: @ma44 and @LetterN 

I am fine with this change. Up to you lads if incremental repairs or something else should be used instead.

**Pull request merged**